docs/source/configuration/cors.mdx

```diff
@@ -17,20 +16,23 @@ server:
   #
   cors:
     # Set to false to disallow any origin and rely exclusively on `origins`
-    # Defaults to true
-    allow_any_origin: true
+    # (Defaults to true)
```

Comment:
Figured we should show some configurations that deviate from the defaults while still showing the defaults